test/worldwide/region_test.rb

```diff
@@ -350,5 +350,49 @@ class RegionTest < ActiveSupport::TestCase
         assert_equal expected_value, Worldwide.region(code: region_code).building_number_required
       end
     end
+
+    test "additional_address_fields returns values as expected" do
```

Comment:
I suggest we only test one or two countries instead of enumerating all of them. Otherwise, as the list of supported additional field countries grows, there will be an expectation of updating this list as well.

Reply:
I was torn about this, but you're right we don't want it to keep growing